bot/constants/messages.py

```diff
@@ -1,20 +1,19 @@
 NETWORK_ERROR_MSG = '😱 There was an error while getting data 😱\nEither your internet connection or the node ' \
                     'endpoint is down! '
-NODE_STARTED_SYNCING = "Your sentry node *{}* is syncing with the network...🚧\n" \
-                       "I will notify you when it's done!"
-NODE_FINISHED_SYNCING = "Your sentry node *{}* is fully synced again!👌\n"
-BOT_RESTARTED = 'Hello there!\n' \
-                'Me, your Node Bot of Terra, just got restarted on the server! 🤖\n' \
-                'To make sure you have the latest features, please start ' \
-                'a fresh chat with me by typing /start.'
-HELLO_MESSAGE = 'Hello there! I am your Node Monitoring Bot of the Terra network. 🤖\n' \
-                'I will notify you about changes of your node\'s *Jailed*, *Unbonded* or *Delegator Shares*, ' \
-                'if your *Block Height* gets stuck and if your *Price Feed* gets unhealthy!\n' \
-                'Moreover, I will notify you about finished and new *governance proposals*. ' \
-                'You can also directly *vote* on them using Terra Station Chrome widget!' \
-                'If you provided IP to your sentry node - I will tell you about all changes of it\'s syncing status'
-
-BOT_STARTUP_LOG = """
+NODE_STARTED_SYNCING_MSG = "Your sentry node *{}* is syncing with the network...🚧\n" \
+                           "I will notify you when it's done!"
+NODE_FINISHED_SYNCING_MSG = "Your sentry node *{}* is fully synced again!👌\n"
+BOT_RESTARTED_MSG = 'Hello there!\n' \
+                    'Me, your Node Bot of Terra, just got restarted on the server! 🤖\n' \
+                    'To make sure you have the latest features, please start ' \
+                    'a fresh chat with me by typing /start.'
+HELLO_MSG = 'Hello there! I am your Node Monitoring Bot of the Terra network. 🤖\n' \
```

Comment:
In the thorchain bot we included a link to the github repo in the start message. 
I think this is pretty neat and could be done here as well.

Reply:
Good idea, I added additional sentence 👍